pkg/apis/extensions/register.go

```diff
@@ -47,6 +47,11 @@ func AddToScheme(scheme *runtime.Scheme) {
 
 // Adds the list of known types to api.Scheme.
 func addKnownTypes(scheme *runtime.Scheme) {
+	// Add known resource names introduced in this API
+	api.AddStandardResourceNames(ResourceDeployments, ResourceReplicaSets)
```

Comment:
@deads2k double check here :-) I feel like this should be in its own function, simil;ar to addKnownTypes (addKnownResourceNames?)